src/changes/applyChanges.ts

```diff
@@ -136,16 +110,10 @@ export function applyAcceptedRejectedChanges(
         },
         node.marks
       )
-      addAttrLog(node.attrs.id, change.dataTracked.id)
     } else if (ChangeSet.isReferenceChange(change)) {
-      const attrs = { ...node.attrs, dataTracked: null }
-      tr.setNodeMarkup(
-        from,
-        undefined,
-        { ...attrs, dataTracked: getUpdatedDataTracked(node.attrs.dataTracked, change.id) },
-        node.marks
-      )
+      tr.setNodeMarkup(from, undefined, { ...node.attrs, dataTracked: null }, node.marks)
```

Comment:
that will drop all track attributes so if the node was splitted multiple times will drop node_split attribute. we need to revert it back

Reply:
I would say that with removing rejected/approved we don't need to care for preserving multiple changes but we can have a look at it later. Reverted

Reply:
when the reference node itself is has a split node it will be important to return it back as it was, so with dropping attr will lose track. also if the reference node is in pending inserted/deleted will need to keep that info for the user